azure_arc_k8s_jumpstart/cluster_api/gitops/basic/az_k8sconfig_capi.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+# <--- Change the following environment variables according to your Azure service principal name --->
+
+echo "Exporting environment variables"
+export appId='<Your Azure service principal name>'
+export password='<Your Azure service principal password>'
+export tenantId='<Your Azure tenant ID>'
+export resourceGroup='<Azure resource group name>'
+export arcClusterName='<The name of your k8s cluster as it will be shown in Azure Arc>'
+export appClonedRepo='<The URL for the "Azure Arc Jumpstart" forked GitHub repository>'
+export namespace='hello-arc'
+
+
+# Installing Helm 3
+echo "Installing Helm 3"
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+chmod 700 get_helm.sh
+./get_helm.sh
+
+# Installing Azure CLI & Azure Arc Extensions
+echo "Installing Azure CLI"
+curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+
```

Comment:
```suggestion

# Installing required Azure Arc CLI extensions
```

Reply:
Removed Azure Arc Extensions from line 21 and added this